python/screenshot-website/README.md

```diff
@@ -0,0 +1,71 @@
+# 📷 Screenshot API
+
+A Python Cloud Function to take a screenshot of an API
+
+_Example input:_
+
+```json
+{
+    "env": 
+    { 
+        "API_FLASH_API_KEY": "<YOUR KEY>" 
+    },
+    "payload": 
+    { 
+        "url": "https://www.bruteforced.dev/" 
+    }
+}
+```
+
+_Example output:_
+
+
+```json
+{
+    "success": true,
+    "screenshot": "iVBORw0KGgoAAAANSUhEUgAAAaQAAALiCAY...QoH9hbkTPQAAAABJRU5ErkJggg=="
+}
+```
+
+>Note that the input JSON has to be a string when embedding as a part of the request's body. Here's what that looks like if you're using a tool like Postman or even cURL
+
+```json
+{
+    "env": "{ \"API_FLASH_API_KEY\": \"<YOUR KEY>\" }",
+    "payload": "{ \"url\": \"https://www.bruteforced.dev/\" }"
+}
+```
+
+>Also, please make sure to replace `YOUR KEY` with the environment variable mentioned below.
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Would you please remove references to "environment" as those are for the older version of open runtimes:

```suggestion
## 📝 Variables

List of variables used by this cloud function:
```